### PR TITLE
DOCSP-23337 users and roles not synched

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-23337-users-and-roles-synched-v0.9.0/reference/mongosync/#cluster-independence)

[JIRA](https://jira.mongodb.org/browse/DOCSP-23337)

### CODE REVIEW

File: source/reference/mongosync.txt [hunk 1]
```diff
@@ -121,6 +121,14 @@ Behavior
 
 .. _c2c-mongosync-config:
 
+Cluster Independence
+~~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` synchronizes collection data between a source cluster and
```
Comment (jason-price-mongodb) [inline source/reference/mongosync.txt:127]: "between a source and destination cluster"

**Review by jason-price-mongodb — APPROVED** [review]
@davemungo LGTM with suggestion. Cheers.

File: source/reference/mongosync.txt (hunk 1 above)
Comment (davemungo) [reply source/reference/mongosync.txt:127]: Done

**Review by davemungo — COMMENTED** [review]
Thanks for the feedback @jason-price-mongodb 

Sending on to external review next.

**Comment by davemungo** [comment]
@tfogo would you review this docs update when you have a moment?

**Review by tfogo — APPROVED** [review]
LGTM!

**Pull request merged**